roles/sap_hana_preconfigure/README.md

```diff
@@ -6,12 +6,18 @@ and then role sap_hana_preconfigure.
 
 ## Requirements
 
-To use this role, your system needs to be configured with the basic requirements for SAP NetWeaver or SAP HANA. This is typically done by running
-role sap_general_preconfigure (for RHEL managed nodes before RHEL 7.6, community maintained role sap-base-settings can be used).
-It is also strongly recommended to run role linux-system-roles.timesync for all systems running SAP HANA, to maintain an identical system time,
-before or after running role sap_hana_preconfigure.
+The role requires additional collections which are specified in `meta/collection-requirements.yml`. These are not automatically installed.
```

Comment:
Suggest overly verbose statement.


This Ansible Role has a dependency requirement on other Ansible Collections, which are specified in `meta/collection-requirements.yml`. These are not automatically installed and are necessary before execution of this Ansible Role, such as installation of the requirements from Ansible Galaxy using:
`ansible-galaxy install -vv -r meta/collection-requirements.yml`

Reply:
Solved in https://github.com/sap-linuxlab/community.sap_install/pull/224/commits/a1662546383075fe18f2303391e0fe7d80d82aaa .